utils/general.py

```diff
@@ -53,6 +54,25 @@ def download_file_blob_to_dir(directory, file_blob):
     os.remove(file_destination)
 
 
+def move_downloaded_data_to_single_directory(directories):
```

Comment:
Semantically, having a function name ending in `single_directory` where the argument is called `directories`, is confusing. Try to give names that convey the meaning of the function directly.

If that doesn't make too much sense, instead try to document what the functionality is.

Reply:
Fixed c24645245cae0f091301b47eb829857fa13fca68